modin/experimental/engines/omnisci_on_ray/frame/partition.py

```diff
@@ -43,6 +43,9 @@ def get(self):
             return self.arrow_table
         return ray.get(self.oid)
 
+    def wait(self):
+        ray.wait([self.oid])
```

Comment:
Why don't we handle ray operations here as we do for pandas on ray? (see [here](https://github.com/modin-project/modin/pull/2801/files#diff-2df6c73d4a63d1c1345522920436cf6b0aec9abd1105fee23ce74a524981a35aR90))

Reply:
I did in the same style as `get` function in `OmnisciOnRayFramePartition` class.

Reply:
We probably need to do the same handling for `OmnisciOnRayFramePartition`'s operations that include ray calls.

Reply:
https://github.com/modin-project/modin/issues/2837